### PR TITLE
slightly update README

### PR DESCRIPTION
**Pull request by HidenoriMatsubayashi**
Update README.

@Mai-Matsuura 
May I request you to review?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -117,15 +117,15 @@ $ make
 
 ### Build for Wayland backend (weston desktop-shell)
 
+This binary will run as a desktop-shell by setting `weston.ini` when Weston starts. See [Settings of weston.ini file](#4-settings-of-westonini-file-only-when-you-use-weston-desktop-shell).
```
Comment (Mai-Matsuura) [inline README.md:120]: The reference is incorrect.  
`(#4-settings-of-westonini-file-only-when-you-use-weston-desktop-shell)`  ->  `(#5-...)`

**Review by Mai-Matsuura — COMMENTED** [review]
I commented. Please check.

**Comment by Mai-Matsuura** [comment]
> Update README.
> 
> @Mai-Matsuura
> May I request you to review?

**Comment by HidenoriMatsubayashi** [comment]
Why did you close the issue? Can you re-open it?

File: README.md (hunk 1 above)
Comment (HidenoriMatsubayashi) [reply README.md]: I fixed the link.

**Comment by Mai-Matsuura** [comment]
> Why did you close the issue?

I'm sorry, I closed the issue by mistake.

**Pull request merged**